package.json

```diff
@@ -3,14 +3,15 @@
   "private": true,
   "version": "3.5.7",
   "description": "🐻 Bear necessities for state management in React",
-  "main": "index.js",
-  "module": "esm/index.js",
-  "types": "index.d.ts",
+  "main": "./index.js",
+  "module": "./esm/index.js",
+  "types": "./index.d.ts",
   "files": [
     "**"
   ],
   "exports": {
     "./package.json": "./package.json",
+    "./": "./",
```

Comment:
https://nodejs.org/api/packages.html#packages_subpath_folder_mappings
says it's deprecated. @barelyhuman

Reply:
we'll add the glob pattern and test that out as well

Reply:
Doesn't the glob pattern conflict with conditional exports? Yeah, testing would be good. I just wonder if there are any environments/versions that don't understand conditional exports but does understand glob patterns. `"./": "./"` was for fallback for old ones, right?

Reply:
> "./" was for fallback for old ones, right? 

based on reading and referencing a few repos on github, yes. 
We could actually remove it and just wait for someone to bring up the issue instead of over thinking it

Reply:
>  just wait for someone to bring up the issue

Yeah, I prefer that tbh.